pkg/helpers/resource_identifier_helpers.go

```diff
@@ -0,0 +1,18 @@
+package helpers
```

Comment:
Nit `helpers` is not needed in filename, because it resides in `helpers` package anyway.

Reply:
Changed